### DISCUSSION TITLE
is Helvetica -> Comic Sans a running joke?

### DISCUSSION THREAD

**Discussion by sherhy** [monkeytypegame/monkeytype#5127] [Questions]
Hi - new to monkeytype, I was just confused in the site configs. There seems to have been a april fools with mixing up the two fonts? Anyhow, is this left here with intention?

https://github.com/monkeytypegame/monkeytype/blob/c3f7f0de6a4299299be5e9f5143d4ff720dd6f32/frontend/static/fonts/_list.json#L36
https://github.com/monkeytypegame/monkeytype/blob/c3f7f0de6a4299299be5e9f5143d4ff720dd6f32/frontend/static/fonts/_list.json#L37

**Comment by Miodec** [comment]
Its a joke, yes.

**Comment by hexxt-git** [comment]
how do i disable it lol

**Reply by fehmer** [reply]
You can use `settings->appearance->font family->custom` and set it to `Helvetica` if you have the font installed locally.
